### PR TITLE
add documentation for template.openshift.io/bindable

### PR DESCRIPTION
**Pull request by jim-minter**
3.7.  After https://github.com/openshift/origin/pull/17215 has merged.
@bmcelvee @bparees

### CODE REVIEW

File: dev_guide/templates.adoc [hunk 1]
```diff
@@ -633,6 +633,19 @@ namespace the parameter substitution resolved the value to, assuming the user
 has permission to create objects in that namespace.
 ====
 
+[[writing-marking-templates-as-bindable]]
+
+=== Marking Templates as Bindable
+
+The template service broker advertises one service in its catalogue for each
```
Comment (bparees) [inline dev_guide/templates.adoc]: s/catalogue/catalog/  (sorry)

**Review by bparees — CHANGES_REQUESTED** [review]

**Comment by bparees** [comment]
one nit and lgtm.

**Comment by jim-minter** [comment]
nit fixed, thanks

**Comment by bmcelvee** [comment]
Thanks, @jim-minter! Looks like has merged, so I'll move forward with this PR. Label for enterprise-3.7, dedicated (3.7), and online (3.7)?

**Comment by jim-minter** [comment]
@bmcelvee sgtm!

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../dev_guide/templates.adoc#dev-guide-templates[Templates]
|Added xref:../dev_guide/templates.adoc#writing-marking-templates-as-bindable[Marking Templates as Bindable] section.
%

**Pull request merged**